android/modules/ui/src/java/kankan/wheel/widget/WheelView.java

```diff
@@ -589,7 +589,13 @@ protected void onDraw(Canvas canvas) {
 		if (itemsWidth > 0) {
 			canvas.save();
 			// Skip padding space and hide a part of top and bottom items
-			canvas.translate(PADDING, -getItemOffset());
+			float translateY;
+			if (Build.VERSION.SDK_INT < 21) {
+				translateY = -getItemOffset();
+			} else {
+				translateY = 0;
```

Comment:
Maggie, this UI widget still has offset issues on API Levels > 21, right? I think we should by translating by the value returned by `getTextSize()` for the higher API Levels. This is because we offset it by text size [here](https://github.com/maggieaxway/titanium_mobile/blob/7765a7d10f51b0f422cd20c54266c2e185eafa73/android/modules/ui/src/java/kankan/wheel/widget/WheelView.java#L362).

That is, it should look like this...
```java
if (Build.VERSION.SDK_INT < 21) {
	translateY = -getItemOffset();
} else {
	translateY = getTextSize() / 2.0f;
}
```

Can you give the above a go please?